src/Http/Http/src/Internal/DefaultHttpRequest.cs

```diff
@@ -10,10 +10,13 @@
 using Microsoft.AspNetCore.Routing;
 using Microsoft.Net.Http.Headers;
 
-namespace Microsoft.AspNetCore.Http.Internal
+namespace Microsoft.AspNetCore.Http
 {
-    public sealed class DefaultHttpRequest : HttpRequest
+    internal sealed class DefaultHttpRequest : HttpRequest
```

Comment:
I feel like we should leave these public?

Reply:
While changing namespaces?

Reply:
Yep.

Reply:
It feels odd to me to expose the DefaultHttpContext but not the rest of the types. Maybe I'm just being picky.

Reply:
Why? They aren't used anywhere, nor very useful in isolation.

Reply:
I found one usage in our entire repo and it was unnecessary. We can bring them back if we find a scenario for them.

Reply:
Why this was made internal? We are using it in our tests where for example have an extension method on HttpRequest and want to test it without to create an entire TestHost.

Reply:
You can still make a DefaultHttpContext to get HttpRequest.

Reply:
Sure, thanks!